src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -1999,10 +1999,10 @@ static void M()
             var compilation = CreateCompilationWithMscorlib(text);
             var tree = compilation.SyntaxTrees.Single();
             var model = compilation.GetSemanticModel(tree);
-            var position = text.IndexOf("{", text.IndexOf("a)"));
+            var position = text.IndexOf('{', text.IndexOf("a)", StringComparison.Ordinal));
```

Comment:
That's fine.  I'm just uncomfortable with us making blanket statements about perf without bench marks to back it up.  There is no way for customers or really anyone to challenge / investigate the statement.